src/editor/guards.ts

```diff
@@ -54,7 +55,21 @@ export const isDateTimeOperator = (
   return propertyType === QueryEditorPropertyType.DateTime && typeof operator?.value === 'string';
 };
 
-export const isMultiOperator = (operator?: QueryEditorOperator): operator is QueryEditorOperator<string[]> => {
+export const isStringArray = (value: any): value is string[] => {
```

Comment:
This could be simplified slightly to be:

```
if (!value instanceof Array) return false;
if (value.every(item => typeof item !== 'string') return false;
return true;
```